config/etc/nginx/nginx.conf

```diff
@@ -0,0 +1,57 @@
+#Template from https://docs.docker.com/registry/recipes/nginx/
+
+events {
+    worker_connections  1024;
+}
+
+http {
+  ## Set a variable to help us decide if we need to add the
+  ## 'Docker-Distribution-Api-Version' header.
+  ## The registry always sets this header.
+  ## In the case of nginx performing auth, the header is unset
+  ## since nginx is auth-ing before proxying.
+  map $upstream_http_docker_distribution_api_version $docker_distribution_api_version {
+    '' 'registry/2.0';
+  }
+
+  server {
+    listen 443 ssl;
+    server_name docker.cdot.systems #${DOMAIN};
```

Comment:
Let's remove the comment, and just hard-code this value.  It's not going to change often enough to bother with the extra machinery to rotate it based on env vars.